src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCodeV1.java

```diff
@@ -65,16 +65,18 @@ public enum ErrorCodeV1 {
 
   SHRED_BAD_BINARY_VECTOR_VALUE("Bad binary vector value to shred"),
 
-  SHRED_BAD_DOCUMENT_TYPE("Bad document type to shred"),
-
   SHRED_BAD_DOCID_TYPE("Bad type for '_id' property"),
 
+  SHRED_BAD_DOCID_EMPTY_STRING("Bad value for '_id' property: empty String not allowed"),
+
+  SHRED_BAD_DOCUMENT_PATH_CONFLICTS("Bad document to shred, contains conflicting Field path(s)"),
```

Comment:
Using V1 errors just because everything in Collection does, easier to convert to v2 as separate step.